test/built-ins/Temporal/Calendar/prototype/fields/repeated-throw.js

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2021 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.calendar.prototype.fields
+description: >
+  Temporal.Calendar.prototype.fields will take iterable of any size and any string
```

Comment:
Maybe something like "Temporal.Calendar.prototype.fields will throw if its input iterable yields the same value twice"?